Sources/WordPressData/Swift/PostCategory+CoreDataClass.swift

```diff
@@ -0,0 +1,19 @@
+import Foundation
+import CoreData
+
+@objc(PostCategory)
+public class PostCategory: NSManagedObject {
+
+    @objc public override class func entityName() -> String {
+        return "Category"
+    }
+
+    @objc public static var uncategorizedId: NSNumber {
```

Comment:
I'm not too sure about `uncategorizedId` but I understand why it works this way – doing `PostCategory.uncategorized` would imply that it's a `PostCategory` _object_ not the default ID.

Probably not worth addressing?